inc/functions.php

```diff
@@ -2178,6 +2178,24 @@ function ppom_get_price_table_calculation() {
 	return apply_filters( 'ppom_price_table_script', $js_script );
 }
 
+/**
+ * Get price table JS dependencies.
+ *
+ * @return string[]
+ */
+function ppom_get_price_table_js_dependencies() {
+	$dependencies = array( 'jquery', 'ppom-inputs' );
+
+	// @phpstan-ignore-next-line
```

Comment:
Here, I assume that PHPStan complains about not finding `WC_VERSION`. In this case, we can create a special file & folder where we add those, like in https://github.com/Codeinwp/feedzy-rss-feeds/blob/master/tests/php/static-analysis-stubs/feedzy.php